src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.inner,
+.wrapper,
+p {
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.wrapper {
+  width: 328px;
+  background-color: #308dc0;
+  margin: 20px;
```

Comment:
Try to resolve your problem only with this margin.
Without pseudo-class.
Hint: `margin: 0 0 0 0;`